dial_worker.go

```diff
@@ -78,8 +79,8 @@ func (w *dialWorker) loop() {
 	defer w.wg.Done()
 	defer w.s.limiter.clearAllPeerDials(w.peer)
 
-	triggerNow := make(chan struct{})
-	close(triggerNow)
+	ready := make(chan struct{})
```

Comment:
What does `ready` do? Why is is closed immediately? Can we have a comment plz?

Reply:
it's a channel signalling immediate action in the loop; but yes, can haz comment.

Reply:
added comment.